### PR TITLE
[element 2.0] Add docs for 'chrome' support, v2.0 changelog, deprecated and missing API

### PR DESCRIPTION
**Pull request by terryytran**

### CODE REVIEW

**Review by phileba — APPROVED** [review]

**Comment by phileba** [comment]
Please try to fix Codacy check errors before merging.

File: packages/docs/docs/api/ElementHandle.md [hunk 1]
```diff
@@ -126,11 +126,11 @@ getProperty
 - key `string`
 - returns: [`Promise<string | null>`][promise]
 
-### `highlight()`
+### ~~`highlight()`~~ DEPRECATED since version 2.0
```
Comment (HieuLeKMS) [inline packages/docs/docs/api/ElementHandle.md:129]: I'm pretty sure `ElementHandle.highlight()` is not deprecated

File: packages/docs/docs/api/Browser.md [hunk 2]
```diff
@@ -200,14 +200,21 @@ Returns the Media (MIME) Type of a file
 - filePath `string` path to a file
 - returns: `string` media type of the file
 
-### `highlightElement(element)`
+### `getUrl()`
 
-Highlight an element. Useful in concert with takeScreenshot to tweak your locators.
+Returns the URL of the current page as a string
```
Comment (HieuLeKMS) [inline packages/docs/docs/api/Browser.md:205]:
```suggestion
Returns the URL of the current page
```

File: packages/docs/docs/start/changelog.md [hunk 3]
```diff
@@ -3,6 +3,32 @@ id: changelog
...
+## 2.0.0
+Released: Feb 2021
+
+### New features
+- Renovate the CLI to be more professional and user-friendly, for a better developer experience.
+- Replace Puppeteer core with Microsoft's Playwright.
+- Support running tests on different browsers including Chromium, Chrome, Firefox, and Webkit with [browser](../guides/test-settings#browser) setting or [`--browser`][cli-run] CLI flag.
+- Support running tests with a custom version of Microsoft Edge, Google Chrome or any Chromium-based browsers by *executablePath* property in [browserLaunchOptions](../guides/test-settings#browserlaunchoptions) setting or [`--executablePath`][cli-run] CLI flag.
+- Support [simulating multiple users on the local machine](../guides/test-settings#stages), with the ability to ramp up or down.
+- Generate an HTML report of the final test results with [`--mu`][cli-run] CLI flag.
```
Comment (HieuLeKMS) [inline packages/docs/docs/start/changelog.md:15]: `--mu` is for simulating multiple users
```suggestion
- Generate an HTML report of the final test results with [`--export`][cli-run] CLI flag.
```

File: packages/docs/versioned_docs/version-2.0/start/changelog.md [hunk 4]
```diff
@@ -3,6 +3,32 @@ id: changelog
...
+## 2.0.0
+Released: Feb 2021
+
+### New features
+- Renovate the CLI to be more professional and user-friendly, for a better developer experience.
+- Replace Puppeteer core with Microsoft's Playwright.
+- Support running tests on different browsers including Chromium, Chrome, Firefox, and Webkit with [browser](../guides/test-settings#browser) setting or [`--browser`][cli-run] CLI flag.
+- Support running tests with a custom version of Microsoft Edge, Google Chrome or any Chromium-based browsers by *executablePath* property in [browserLaunchOptions](../guides/test-settings#browserlaunchoptions) setting or [`--executablePath`][cli-run] CLI flag.
+- Support [simulating multiple users on the local machine](../guides/test-settings#stages), with the ability to ramp up or down.
+- Generate an HTML report of the final test results with [`--mu`][cli-run] CLI flag.
```
Comment (HieuLeKMS) [inline packages/docs/versioned_docs/version-2.0/start/changelog.md:15]:
```suggestion
- Generate an HTML report of the final test results with [`--export`][cli-run] CLI flag.
```

**Review by HieuLeKMS — APPROVED** [review]

**Pull request merged**